docs/source/guides/tools/rpakpilotmodding.rst

```diff
@@ -0,0 +1,245 @@
+Use LegionPlus to view paths for materials. Scroll below a little bit to learn how to use Legion+.
+
+.. image:: /_static/importpilotmodel/ss2-pilotBlenderMatl.png
+   :align: center
+   :class: screenshot
+
+Once you have made all the changes you wanted to, select ``Scene Properties``, ``Source Engine Export``, then set the export format to ``SMD`` and set your export path.
+
+.. image:: /_static/importpilotmodel/ss3-BlenderQcModifier.png
+   :align: center
+   :class: screenshot
+
+Copy the qc file to the same location where you exported the model.
+
+Modifying the POV model
+^^^^^
+
+- Select ``File`` in Blender, then select ``Save Copy``
+- Choose any name, save the copy, then open the copy
+- Delete everything except for the body, the body collection, and the skeleton
+- Add ``pov_`` to the beginning of your body model and armature
+- Repeat the process of exporting the model with ``Source Engine Export``
+
+Using Legion+ to view paths
+^^^^^
+
+Open `Legion+`_
+-----
+
+Select ``Load File``
+Follow the directory that matches your launcher.
+
+1. Steam: ``Steam\steamapps\common\Titanfall2\r2\paks\Win64\common.rpak`` or
+2. Origin: ``Origin Games\Titanfall2\r2\paks\Win64\common.rpak``
+
+Type the file name of the pilot into the search bar to view **texture and material paths**.
+View file paths to create paths in RePak
+
+Recompiling the model
+^^^^^
+
+Install compatible Source game
+----
+
+You need a compatible Source game installed when using Crowbar.
+Compatible Source games in order of preference:
+
+Compatible Source games
+#####
+
+1. Portal 2
+2. Alien Swarm
+3. Source FilmMaker
+
+Open `Crowbar`_
+-----
+
+Select ``Compile``
+Click on the Browse button next to ``QC input`` then locate and select your modified model.
+Next, Click on the Browse button next to ``Output to`` then choose a location to output the model to.
+
+Making a Pak file
+^^^^^
+
+Follow this `tutorial <https://r2northstar.readthedocs.io/en/latest/guides/rpakmodding.html>`_ to create a pak file
+
+Making changes to the qc file (Optional)
+^^^^^
+
+Open any text editor 
+If you want to prevent camos from being used on your model, delete any textures that have skn31 in the texture group
+
+.. figure:: /_static/importpilotmodel/ss4-qcFileChanges.png
+   :align: center
+
+.. figure:: /_static/importpilotmodel/ss5-qcFileChanges2.png
+    :align: center
+
+Creating a Northstar mod
+^^^^^
+
+Follow the "Getting Started" guide to create a Northstar mod
+Create a folder that matches the below file structure
+
+.. note:: Only create the ``weapons`` folder with everything else inside it if you made a pov model
+
+.. Directory Structure for Northstar Mod
+:: 
+
+    AuthorName.Mod
+    ├──mod
+    |   └──models
+    |       └──humans
+    |           └──pilots
+    |               └──customModel
+    ├──paks
+    |   ├──rpak.json
+    |   ├──example.rpak
+    |   └──preloadexample.rpak
+    └──mod.json
+
+Credits: 
+^^^^^
+
+Base guide from Harb. 
+Edited by Northstar.
```

Comment:
Any chance the base guide could be linked here?

Reply:
Sure.

I'll add the original google link, and experiment with the file sharing on the docs.